deploy/charts/klicker-uzh-v2/values.yaml

```diff
@@ -248,5 +248,57 @@ backendGraphql:
 
   affinity: {}
 
+lti:
+  priorityClassName: production-workload
+
+  replicaCount: 2
+
+  clientId:
+  redirectURL:
+  cookieDomain:
+  encryptionKey:
+
+  db:
+    host:
+    port:
+    name:
+    username:
+    password:
+
+  image:
+    repository: ghcr.io/uzh-bf/klicker-uzh/lti
+    pullPolicy: Always
+
+  service:
+    type: ClusterIP
+    port: 4000
+
+  ingress:
+    enabled: true
+    annotations:
+      nginx.ingress.kubernetes.io/proxy-body-size: 256m
+      cert-manager.io/cluster-issuer: letsencrypt
+
+  resources:
+    requests:
+      cpu: 200m
+      memory: 200Mi
+    limits:
+      cpu: 500m
+      memory: 500Mi
+
+  autoscaling:
+    enabled: true
```

Comment:
Is autoscaling enabled on purpose here, and does it react quickly enough to handle a suddenly increased demand in LTI requests? What happens, if the service is not able to react sufficiently quickly during a surge in demand - are requests dropped and the automated login to OLAT fails?